torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -32,71 +32,76 @@ FunctionSchema PythonValue::getSchema(
     const size_t n_binders,
     const SourceRange& loc) {
   auto annotations = py::module::import("torch.jit.annotations");
-  const auto fn_to_get_signature =
-      moduleSelf_ ? py::getattr(self, "original_fn") : self;
-  auto signature = annotations.attr("get_signature")(
-      fn_to_get_signature, rcb ? *rcb : py::none(), loc);
+  const auto callable = moduleSelf_ ? py::getattr(self, "original_fn") : self;
+
+  // Make sure the function is not a class instantiation
```

Comment:
What does this mean: "class instantiation" ?

Reply:
creating an instance of a class (e.g. `Exception()`), I added a comment to clarify